lang/lang.nl.json

```diff
@@ -0,0 +1,929 @@
+      "donate": "/\\b({{REF:commands.donate}}|d)\\b/i",
+      "find": "/\\b({{REF:commands.find}}|v|zoekOp|zoeken|zoek)\\b/i",
+      "help": "/\\b({{REF:commands.help}}|h|\\?)\\b/i",
+      "info": "/\\b({{REF:commands.info}}|i|informatie)\\b/i",
+      "invite": "/\\b({{REF:commands.invite}}|u|uitnodigen)\\b/i",
+      "list": "/\\b({{REF:commands.list}}|l)\\b/i",
+      "map": "/\\b({{REF:commands.map}}|k|zone|zones|kiezer)\\b/i",
+      "me": "/\\b({{REF:commands.me}}|ik|zelf|gebruiker)\\b/i",
+      "server": "/\\b({{REF:commands.server}}|guild|config|instelling|instellingen)\\b/i",
+      "set": "/\\b({{REF:commands.set}}|z|)\\b/i",
+      "setup": "/\\b({{REF:commands.setup}}|instellen|installeer|installeren|start|starten)\\b/i",
+      "support": "/\\b({{REF:commands.support}}|s)\\b/i",
+      "time": "/\\b({{REF:commands.time}}|bekijk|nu|datum)\\b/i",
+      "translate": "/\\b({{REF:commands.translate}}|vertaling|vertalingen|vertalen|taal|talen)\\b/i",
+      "vote": "/\\b({{REF:commands.vote}})\\b/i",
+      "stop": "/\\b({{REF:commands.stop}}|sluit|sluiten|stoppen|annuleer|annuleren|cancel|exit)\\b/i",
+      "remove": "/\\b({{REF:commands.remove}}|v|wis|reset|verwijderen)\\b/i"
+    },
+    "settings": {
+      "autoDetect": "/\\b({{REF:settings.autoDetect}}|a|ad|auto-detecteer|automatisch|detecteren)\\b/i",
+      "dateFormat": "/\\b({{REF:settings.dateFormat}}|d|df|datumParser|datum|parser)\\b/i",
+      "language": "/\\b({{REF:settings.language}}|t|taalCode)\\b/i",
+      "list": "/\\b({{REF:settings.list}})\\b/i",
+      "privateMode": "/\\b({{REF:settings.privateMode}}|p|prive)\\b/i",
+      "reminders": "/\\b({{REF:settings.reminders}}|h|herinnering|herinner|herinneringenIngeschakeld)\\b/i",
+      "timeFormat": "/\\b({{REF:settings.timeFormat}}|tf|tijd)\\b/i",
+      "timeZone": "/\\b({{REF:settings.timeZone}}|t|tz|zone)\\b/i"
+    },
+    "dateFormat": {
+      "dayMonth": "/\\b({{REF:dateFormat.dayMonth}}|dm|ddmm|dd-mm|dag-maand|dag/maand)\\b/i",
+      "monthDay": "/\\b({{REF:dateFormat.monthDay}}|md|mmdd|mm-dd|maand-dag|maand/dag)\\b/i"
+    },
+    "timeFormat": {
+      "twentyFourHour": "/\\b({{REF:timeFormat.twentyFourHour}}|24u|24-uur)\\b/i",
+      "twelveHour": "/\\b({{REF:timeFormat.twelveHour}}|12u|12-uur)\\b/i"
+    },
+    "yesNo": {
+      "yes": "/\\b({{REF:yesNo.yes}}|j|ingeschakeld|inschakelen|i|true|t|aan|1)\\b/i",
+      "no": "/\\b({{REF:yesNo.no}}|n|uitgeschakeld|uitschakelen|u|false|f|uit|0)\\b/i"
+    }
+  },
+  "refs": {
+    "meta": {
+      "language": "nl",
+      "languageDisplay": "Nederlands",
+      "locale": "nl",
+      "translators": "[Hester#5259](https://github.com/uxster)"
+    },
+    "bot": {
+      "name": "Friend Time",
+      "description": "{{REF:bot.name}} helpt het coördineren van tijden en evenementen in jouw server door tijden in chatberichten te converteren naar verschillende tijdzones!",
+      "prefix": "-ft"
+    },
+    "colors": {
+      "success": "00ff83",
```

Comment:
Please add hashtag symbols with the colors here. This was another change I made in master:
```json
"colors": {
    "success": "#00ff83",
    "warning": "#ffcc66",
    "error": "#ff4a4a"
},
```